### PR TITLE
CODECONVENTIONS: Clarify MicroPython changes sign-off process.

### PR DESCRIPTION
**Pull request by pfalcon**
In particular, require the real name and email address.

### CODE REVIEW

**Comment by pfalcon** [comment]
Better late than never.

File: CODECONVENTIONS.md [hunk 1]
```diff
@@ -26,6 +26,25 @@ require such detailed description.
...
+  from and give credit to the original author, as required by the license).
+* That you are allowed to release these changes to an open-source project
+  (for example, changes done during paid work for a third party may require
+  explicit approval from that third party).
+* That you (or your employer) agree to release the changes under
+  MicroPython's license, which is the MIT license. Note that you retain
+  copyright for your changes (for smaller changes, the commit message
+  conveys your copyright; if you make significant changes to a particular
+  source module, you're welcome to add your name to the file header).
+* Your signature for all the above, which should include the full real
```
Comment (dpgeorge) [inline CODECONVENTIONS.md]: "Your signature for all of the above, which is the 'Author' line in the commit message, and which should include your full real name and a valid and active email address by which you can be contacted in the foreseeable future."

Comment (pfalcon) [inline CODECONVENTIONS.md]: Updated

**Comment by dpgeorge** [comment]
Looks good, thanks.

**Comment by pfalcon** [comment]
Merged.

**Pull request closed without merging**